content/overview/security/for-assessors.md

```diff
@@ -0,0 +1,59 @@
+---
+menu:
+  overview:
+    parent: technology
+title: Overview for assessors
+weight: 50
+aliases:
+  - /docs/intro/security/for-assessors
+  - /intro/security/for-assessors
+  - /overview/security/for-assessors/
+---
+
+## How cloud.gov works
+
+cloud.gov is a platform as a service. This means customers provide the code for their app and we provide the infrastructure, operating system, and supporting technologies required to run the app. There are three basic components to any cloud.gov app:
```

Comment:
Do you mean "three basic components _underneath_ any cloud.gov app"?

Reply:
@gboone ^